packages/venia-ui/lib/components/MiniCart/miniCart.gql.js

```diff
@@ -1,14 +1,17 @@
 import gql from 'graphql-tag';
 
 import { ProductListFragment } from './ProductList/productList.gql';
+import { PriceSummaryFragment } from '../CartPage/PriceSummary/priceSummaryFragments';
 
 export const MiniCartFragment = gql`
     fragment MiniCartFragment on Cart {
         id
         total_quantity
         ...ProductListFragment
+        ...PriceSummaryFragment
```

Comment:
Will need to refetch the entire cart/checkout page fragments, not just the price summary.

Shipping values change, overall cart value can change and go to $0 if there is a GC covering some of the cost, etc.

Reply:
I purposefully didn't add the GC, Coupons, etc because those won't change anyway based on the cart. But you might be on to something here. To avoid weird edge cases, it is better to fetch the whole cart fragment.

Do we need to have the mini cart fragment same as the cart fragment?

Reply:
Updated in 3a399cd